ui-ngx/src/app/modules/home/components/widget/lib/multiple-input-widget.component.ts

```diff
@@ -362,6 +368,9 @@ export class MultipleInputWidgetComponent extends PageComponent implements OnIni
             case 'select':
               value = keyData[0][1].toString();
               break;
+            case 'JSON':
+              value = isString(keyData[0][1]) ? JSON.parse(keyData[0][1]) : keyData[0][1];
+              break;
```

Comment:
We need to use try-catch to parse JSON because the widget may stop working due to an invalid JSON.